hack/build/ci/create_go_build_args.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+version=$1
```

Comment:
could use checks if the parameters are not empty/exist

Reply:
implemented